### PR TITLE
@mux/mux-node update

### PR DESCRIPTION
**Pull request by dylanjha**
```
npx @getgrit/launcher apply mux_v8
```

### CODE REVIEW

File: pages/api/webhooks/mux.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const webhookSignatureSecret = process.env.MUX_WEBHOOK_SIGNATURE_SECRET;
 const verifyWebhookSignature = (rawBody: string | Buffer, req: NextApiRequest) => {
   if (webhookSignatureSecret) {
     // this will raise an error if signature is not valid
-    Mux.Webhooks.verifyHeader(rawBody, req.headers['mux-signature'] as string, webhookSignatureSecret);
+    Mux.Webhooks.prototype.verifySignature(Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers, webhookSignatureSecret)
```
Comment (dylanjha) [inline pages/api/webhooks/mux.ts:14]: Looks like it didn't add

`const mux = new Mux()`

and it went with `Mux.Webhooks.prototype.verifySignature`, which I don't think is right

Comment (dylanjha) [reply pages/api/webhooks/mux.ts:14]: Also interesting that it added this ternary, which wasn't there before:

```
Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers
```

Comment (rattrayalex) [reply pages/api/webhooks/mux.ts:14]: cc @morgante can you take a look at this?

Comment (rattrayalex) [reply pages/api/webhooks/mux.ts:14]:
```ts
const mux = Mux() // maybe earlier in the file
mux.webhooks.verifyHeader(Buffer.isBuffer(rawBody) ? rawBody.toString('utf8') : rawBody, req.headers, webhookSignatureSecret)
```

**Comment by rattrayalex** [comment]
Ah, that build error is likely because we marked properties as required in `NewAssetSettings` despite them being request params – @jsanford8 , does that sound right? Could you get that fixed in the OpenAPI Spec?

**Pull request closed without merging**